pkg/auth/providers/saml/saml_provider.go

```diff
@@ -115,6 +115,12 @@ func PerformSamlLogin(name string, apiContext *types.APIContext, input interface
 	finalRedirectURL := login.FinalRedirectURL
 
 	if provider, ok := SamlProviders[name]; ok {
+		if provider == nil {
```

Comment:
Do we just need to wait longer until it's initialized? And where is this error going to show up?

Reply:
So this has been reported intermittently during login, a while after auth was configured, so I don't think we can add wait logic. I think this error should show up on the login page, but now I've also added it as an error log